content/docs/guides/incremental-datasets.md

```diff
@@ -111,6 +111,26 @@ This dataset may not exist in the warehouse yet. That's OK, because the `WHERE`
   Dataform Web.
 </div>
 
+## A merge modification
+
+<div className="bp3-callout bp3-icon-info-sign" markdown="1">
+  Incremental merging requires <code>@dataform/core</code> version <code>1.4.21</code>.<br>
```

Comment:
As discussed in DD, we shouldn't just change this behavior. We should only apply the new logic if the user is on a version >= 1.4.21.

That means gating all the logic for merge statements in the adapter SQL generation code based on the version I'm afraid :(

Reply:
I'm confused, isn't this what I've done here already?

Reply:
Sorry weird place to put comment. - The DD suggested we only use the new behaviour (in code) if the user updates.

However, now that we are using the old insert behaviour by default, this release should be a no-op for all existing projects, and we only use merge once they set a uniqueKey value.

So, ignore my comment, we can leave as is :)